### ISSUE TITLE
Alert info: i ball doesn't pass colour contrast test

### ISSUE THREAD

**Issue by XiuJuanXia** [wet-boew/cdts-sgdc#256]
In the alert info, the i ball doesn't pass colour contrast test. The current contrast ratio is 2.956:1. And the minimum contrast ratio should be 3:1 based on WCAG 2.1 1.4.11 Non-text Contrast. This is an accessibility issu.e
Foreground colour: #D7FAFF
Background colour: #269ABC

Example pages that have the alert info:
https://cenw-wscoe.github.io/sgdc-cdts/docs/index-en.html
https://cenw-wscoe.github.io/sgdc-cdts/docs/contact-en.html
![image](https://user-images.githubusercontent.com/88776404/131514349-73bde383-b3b9-4579-99cc-e58ed7c5626f.png)

Please refer to https://wet-boew.github.io/GCWeb/components/components-en.html which has the most recent info alert.

**Comment by ahmad-shahid** [comment]
Hello, the alert info has been updated. We are just waiting for deployment.

**Comment by ahmad-shahid** [comment]
The alert has now been updated and is live.

**Comment by ahmad-shahid** [comment]
![image](https://user-images.githubusercontent.com/52757738/139847874-66449705-2e93-4c77-ae3b-ea22fb2b44ca.png)

**Comment by XiuJuanXia** [comment]
Issue is fixed and closing the issue.
